packages/koa-shopify-graphql-proxy/src/shopify-graphql-proxy.ts

```diff
@@ -37,6 +38,12 @@ export default function shopifyGraphQLProxy(proxyOptions: ProxyOptions) {
       'password' in proxyOptions ? proxyOptions.password : session.accessToken;
     const version = proxyOptions.version;
 
+    if (version == April19 || version == July19) {
```

Comment:
`July19` is valid until July 1, 2020. I also think we should remove `April19` from the enum instead of adding this message.

Reply:
I suggested the same but was told to leave it in, I am doing as I was told in the #api-release-mgmt. If this is an issue, please raise it there.

Reply:
After discussion on slack, you can 🔥 the `April19`